lib/services/parser.go

```diff
@@ -176,6 +176,19 @@ func NewWhereParser(ctx RuleContext) (predicate.Parser, error) {
 				}
 				return string(ca.GetType()), nil
 			},
+			"startsWith": func(a, b interface{}) predicate.BoolPredicate {
```

Comment:
Most of the [functions](https://github.com/gravitational/teleport/blob/97cb4f8dc80ce9451bbe59e8000e4b14cac4c9d6/lib/expression/parser.go#L38-L154) in our predicate language are lowercase, either smooshed together (e.g. `ifelse`) or in snake case (e.g. `contains_all`) and some have prefixes/namespaces (e.g. `strings.lower`).

What do you think of making this `strings.has_prefix`?

Reply:
Yeah, +1 on renaming to something `snake_case`

Reply:
Hm - so I thought I'd chosen startsWith based on the fact we support this in other predicates within Teleport, but it looks like I hallucinated that. I'll switch this out.

Reply:
[501f701](https://github.com/gravitational/teleport/pull/53956/commits/501f701f983da8ee2bf35af03ffde5168a6343e3)